session.go

```diff
@@ -109,6 +111,7 @@ type session struct {
 	store   kv.Storage
 	sid     int64
 	history stmtHistory
+	initing bool
```

Comment:
No, in bootstrap stage, autocommit checker will return false. For mysql.Global_variables table is not created. Access to this table will cause error. (AutocommitChecker should check autocommit variable, which is a system variable.)
So I use initing to indicate we are running bootstrap using this session.